contracts/mock/EIP173Proxy.sol

```diff
@@ -0,0 +1,93 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.7.0;
+
+import "./Proxy.sol";
+
+interface ERC165 {
+    function supportsInterface(bytes4 id) external view returns (bool);
+}
+
+///@notice Proxy implementing EIP173 for ownership management
+contract EIP173Proxy is Proxy {
```

Comment:
Is this meant to implement ERC165? (`is`)?

Reply:
I don't think it does. This is the proxy that hardhat-deploy plugin uses under the hood for deployments.